src/otx/core/model/detection.py

```diff
@@ -96,12 +96,12 @@ def test_step(self, batch: OTXDataBatch, batch_idx: int) -> None:
 
         if isinstance(metric_inputs, dict):
             self.metric.update(**metric_inputs)
-            return
+            return preds
 
         if isinstance(metric_inputs, list) and all(isinstance(inp, dict) for inp in metric_inputs):
             for inp in metric_inputs:
                 self.metric.update(**inp)
-            return
+            return preds
```

Comment:
@itrushkin Thanks for your contribution to this PR! Would you mind updating all tasks (not just detection) to include `return preds` in both `validation_step` and `test_step`, along with adjusting the return type annotation to `OTXPred`? This would help us ensure consistency across the board. Once that's in place, we'll proceed with formulating the VisualizerHook within our team.